### PR TITLE
Explain how to restore a previously created dump

### PR DESCRIPTION
**Pull request by stromvirvel**
Update README.md with info how to restore a dump.

### CODE REVIEW

**Comment by mitar** [comment]
Thanks.

File: README.md [hunk 1]
```diff
@@ -40,6 +40,11 @@ You can backup a database from outside the container:
 $ docker exec postgresql pg_dump -Fc -U postgres <DBNAME> > /var/backups/<DBNAME>.pgdump
 ```
 
+You can restore a database from outside the container:
+```
+$ cat /var/backups/<DBNAME>.pgdump | docker exec -i postgresql pg_restore -Fc -U postgres -d <DBNAME>
```
Comment (mitar) [inline README.md:45]: Why `-i`?

**Comment by mitar** [comment]
BTW, for backing up I made this [image](https://github.com/tozd/docker-rdiff-backup).

**Comment by moepot** [comment]
In my case, the pipe doesn't work without `-i`. Just hangs forever and nothing happens. Maybe you can try by yourself?

File: README.md (hunk 1 above)
Comment (stromvirvel) [reply README.md:45]: In my case, the pipe doesn't work without `-i`. Just hangs forever and nothing happens. Maybe you can try by yourself?

Comment (mitar) [reply README.md:45]: Oh, OK.

**Pull request merged**